src/components/TopBar.js

```diff
@@ -326,15 +286,10 @@ const TopBar = Utils.connectAtom(authStore, 'authState')(class TopBar extends Co
           ]),
           div({ style: { borderTop: `1px solid ${colors.dark(0.55)}` } }, []),
           div({
-            style: {
-              ..._.omit('borderTop', styles.nav.item),
-              marginTop: 'auto',
-              color: colors.dark(0.55),
-              fontSize: 10
-            }
+            style: { flex: 'none', padding: 28, marginTop: 'auto', color: colors.dark(0.55), fontSize: 10, fontWeight: 600 }
           }, [
             'Built on: ',
-            a({
+            h(Clickable, {
```

Comment:
A reasonable point. I guess my thinking was to standardize on `Clickable` as a baseline, since it provides a common place to add default behaviors. FWIW we have very few explicit `a` tags, so it's also a bit of consistency.